### PR TITLE
Travis merge

### PR DESCRIPTION
**Pull request by gidden**
This PR adds Travis support and updates our `FindHDF5.cmake` to be in line with Pyne's. This allows someone to declare a variable `HDF5_ROOT` which should be able to help in finding HDF5. Additional debug logging was added to our build system to that effect.

For successful runs (with a successful failure), see https://github.com/gidden/cyclus/pull/16

This is blocked on https://github.com/cyclus/ciclus/pull/62

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  # install cyclus
+  - ./.travis-install.sh
+
+script:
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - export LD_LIBRARY_PATH="$HOME/miniconda/lib:$LD_LIBRARY_PATH"
+  #- ls -l $HOME/miniconda/lib
+  #- ls -l $HOME/miniconda/envs/_build/lib
+  #- ls -l $HOME/miniconda/include
```
Comment (scopatz) [inline .travis.yml]: Remove comments

File: .travis-install.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+wget https://github.com/cyclus/ciclus/archive/master.zip -O ciclus.zip
+unzip -j ciclus.zip "*/cyclus/*" -d conda-recipe
+
+# build
+cmd="conda build --no-test conda-recipe"
+echo "cmd: $cmd"
+$cmd
```
Comment (scopatz) [inline .travis-install.sh]: It seems like you are reimplementing `set -x`.  Why not just use `set -x`

**Comment by gidden** [comment]
ping, this is ready

**Comment by scopatz** [comment]
Can you take down polyphemus please?  I need to verify that this is working on travis before I can merge.

**Comment by gidden** [comment]
I can't ssh into moth given the password in the metadata file.

On Thu, Apr 23, 2015 at 4:39 PM, Anthony Scopatz <notifications@github.com>
wrote:

> Can you take down polyphemus please? I need to verify that this is working
> on travis before I can merge.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus/pull/1130#issuecomment-95727203>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

**Comment by scopatz** [comment]
are you looking at the "accounts metadata" file?

**Comment by gidden** [comment]
no, it's still in cyclus metadata

**Comment by scopatz** [comment]
I think that cyclus metadata is out of date.

**Comment by gidden** [comment]
RIP polyphemus

**Comment by scopatz** [comment]
<a href="http://s429.photobucket.com/user/williamethridge/media/ed268005621a423188f286e1a90b738334269c6a_zps2affb8c9.png.html" target="_blank"></a>

**Comment by gidden** [comment]
I've started the travis job

**Comment by gidden** [comment]
closes #1117

**Comment by gidden** [comment]
ping @scopatz, travis passes

**Comment by scopatz** [comment]
It seems like my comments about set -x and comment removal still apply.  I thought that you had fixed these though.  Did something funny happen in a rebase?

**Comment by gidden** [comment]
updated

**Comment by scopatz** [comment]
Seems reasonable.  Waiting for CI.

**Comment by gidden** [comment]
it takes ~20 minutes, FYI

**Comment by gidden** [comment]
after you pull, could you relaunch https://travis-ci.org/cyclus/cycamore/builds/59900569, please?

**Comment by scopatz** [comment]
Yeah, I have been looking at the time stamps.  That is about 10 min faster than batlab.

**Comment by scopatz** [comment]
I am not working on cycamore or the release. I am helping out mostly as a transitional crutch.  I have a lot of other things on my critical path right now. Sorry.

**Comment by scopatz** [comment]
Looks good to me. Thanks!

**Comment by scopatz** [comment]
Oh shoot! Wrong gh window!

**Comment by scopatz** [comment]
Will revert if it fails.

**Comment by scopatz** [comment]
74cd389 failed.  Would you mind opening a new PR with a fix.

**Comment by scopatz** [comment]
Sorry @gidden

**Comment by scopatz** [comment]
Seems like some weird Travis issue.

**Comment by gidden** [comment]
It's simply clicking on a link then pressing a button. It's also related to
migrating to travis...

**Comment by scopatz** [comment]
Thanks again, Matt.  I have merged this in via #1132.  Sorry about that wrong github tab nonsense. My bad.

**Pull request merged**